spring-grpc-spring-boot-autoconfigure/src/test/java/org/springframework/grpc/autoconfigure/common/GrpcCodecConfigurationTest.java

```diff
@@ -0,0 +1,82 @@
+package org.springframework.grpc.autoconfigure.common;
+
+import io.grpc.Compressor;
+import io.grpc.CompressorRegistry;
+import io.grpc.Decompressor;
+import io.grpc.DecompressorRegistry;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.autoconfigure.AutoConfigurations;
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.boot.test.context.runner.ApplicationContextRunner;
+import org.springframework.context.annotation.Bean;
+import org.springframework.grpc.autoconfigure.common.codec.GrpcCodecConfiguration;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Tests for {@link GrpcCodecConfiguration}.
+ *
+ * @author Andrei Lisa
+ */
+public class GrpcCodecConfigurationTest {
+
+	private final ApplicationContextRunner contextRunner = new ApplicationContextRunner()
+		.withConfiguration(AutoConfigurations.of(GrpcCodecConfiguration.class));
+
+	@Test
+	void testCompressorRegistryBean() {
+		contextRunner.run(context -> {
+			assertThat(context).hasSingleBean(CompressorRegistry.class);
```

Comment:
The cool thing about `hasSingleBean` is that it does the get bean and ensures there is exactly 1 bean registered. The 2 lines below are redundant and can be removed. 🎉

Reply:
done.